### ISSUE TITLE
Edico add-on incompatibility with NVDA 2024.1

### ISSUE THREAD

**Issue by EmmanuelPulidoTinajero** [edicoIta/edico-nvda#1]
The Edico add-on is currently incompatible with the latest version of NVDA, 2024.1. This issue prevents users from using Edico's features, such as text editing and document navigation, with the latest version of NVDA.
Steps to reproduce:
1. Install NVDA 2024.1.
2. Install the Edico add-on.
3. Restart NVDA.
4. Attempt to use Edico's features.
Expected behavior:
Edico's features should be available and functional in NVDA 2024.1.
Actual behavior:
Edico's features are not available or functional in NVDA 2024.1.
Workaround:
There is currently no workaround for this issue.
Request:
Please update the Edico add-on to be compatible with NVDA 2024.1. This will allow users to continue using Edico's features with the latest version of NVDA.

**Comment by albzan** [comment]
This project seems to be not up-to-date with the latest versions of NVDA.
I forked the addon:
https://github.com/edicoitalia/edico-nvda
I implemented bug fixes and I tested and released the 2024.1 compatible addon on the Add-On store:
https://github.com/nvaccess/addon-datastore/pull/3177
It has been merged on yesterday
